SpatialGDK/Source/SpatialGDKEditor/Public/SpatialGDKEditorSettings.h

```diff
@@ -287,6 +288,45 @@ struct SPATIALGDKEDITOR_API FRuntimeVariantVersion
 	FString PinnedVersion;
 };
 
+USTRUCT()
+struct SPATIALGDKEDITOR_API FInspectorVersion
+{
+	friend class USpatialGDKEditorSettings;
+
+	GENERATED_BODY()
+
+	FInspectorVersion()
+		: PinnedVersion(SpatialGDKServicesConstants::SpatialOSInspectorPinnedStandardVersion)
+	{
+	}
+
+	FInspectorVersion(const FString& InPinnedVersion)
+		: PinnedVersion(InPinnedVersion)
+	{
+	}
+
+	/** Returns the Inspector version to use, either the pinned one, or the user-specified one depending on the settings. */
+	const FString& GetInspectorVersion() const;
+
+	bool GetUseGDKPinnedInspectorVersion() const { return bUseGDKPinnedInspectorVersion; }
+
+	const FString& GetPinnedVersion() const { return PinnedVersion; }
```

Comment:
do we need this? I would expect users to only ever call `GetInspectorVersion`

Reply:
Yeah agree with Jess. There's no need for the struct as it's not like the RuntimeVariants